JobScheduler/GraduationWorker.cs

```diff
@@ -0,0 +1,68 @@
+using MessageBrokerPublisher;
+using System;
+using System.Threading;
+using System.Threading.Tasks;
+using Microsoft.Extensions.Hosting;
+using Microsoft.Extensions.Logging;
+using System.Collections.Generic;
+using Models;
+using NotificationSystem.Notifications;
```

Comment:
This is a problem with how the current notification system is set up, this is an error since the publisher should not know anything about the consumer (notification system). The fault here at hand is that this is now using the model in the notification system but there is no DTO available in the publisher. I've corrected this mistake in PR: https://github.com/DigitalExcellence/dex-backend/pull/344 .
This should use a DTO available in MessageBrokerPublisher/Models .
Bit of a pain, I know, but necessary if we want to follow the principle of the pub/sub pattern which supplies decoupling.